proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,471 @@
+1. The change does not increase the maintainability in an unreasonable fashion.
+1. There is sufficient documentation, both in the code and for end users.
+1. The code is readable and follows best practices and conventions.
+
+If a pull request is not ready to merge, the maintainers will request feedback
+and changes from the contributor by asking specific questions or specifying
+criteria that should be met. If changes are requested, the contributor should
+make the changes within [TBD] business days. If the contributor does not respond
+within [TBD] days, the pull request will be marked as stale and the maintainers
+will complete the pull request.
+
+Once a pull request is completed to the satisfaction of the maintainers, it will
+be approved and merged. The contribution will be labeled to acknowledge it as a
+community contribution.
+
+## Process implementation
+
+The contribution process will be implemented with a publicly visible [GitHub
+project
+board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards).
+This project board will track GitHub issues from the contribution-ready stage
+through completion by merging a pull request. New and existing GitHub labels and GitHub issue
+events will be used to drive the movement of the issue between the lanes of the
+GitHub project, which map to the stages of the contribution process.
+
+The first lane in the GitHub project contains the list of contribution-ready
+issues. This is the entry point for a user interested in making a contribution.
+
+When a contributor opens a pull request for an issue in the contribution ready
+list, it will be moved to the Implementation lane. This indicates that the
+contibution is ready for an initial review by the maintainers. The maintainers
+will perform an initial review within [TBD] days.
+
+Once the initial review is confirmed to be complete, the issue can be labeled as
+ready for review, and it moves to the 'Ready for Review' lane. These issues
+will be prioritized, selected, and labeled for review on a [TBD] basis.
+
+Once the maintainers have selected a pull request for review, a label marking it
+as ready for review will be added, and the pull request will move to the 'Under
+Review' lane of the GitHub project. This lane will provide visibility to the
+community for what the maintainers are currently working on. 
+
+After the review process is completed and a pull request is merged, the issue
+will be closed and moved to the 'Done' lane. This will provide the community
+with a list of successful contributions. 
+
+## Managing the existing backlog
+
+As of 2021-12-15, there are 188 pull requests and 436 issues in the AWS CLI
+GitHub repository. 43 pull requests and 87 issues have not been updated in more
+than a year. 95 issues have ten or more upvotes. [TBD] of the open pull requests
+are directly related to an existing feature request. [TBD] of open pull requests
+have extensive merge conflicts.
```

Comment:
I'm not sure if we need these additional stats. We are probably just fine with the total amounts and saying we need to onboard these issues to the proposed process.